plone_training_config/Vagrantfile

```diff
@@ -22,6 +22,16 @@
   # A workaround for missing symbolic links on windows:
   # config.vm.synced_folder "eggs/", "/home/vagrant/buildout-cache/eggs/"
 
+  # Read initial package index information + upgrade
+  config.vm.provision :shell, inline: <<-SHELL
+      apt-get update
+      apt-get -y upgrade
```

Comment:
isn't that done by aptgetupdate.pp?

Reply:
chicken or egg: `aptgetupdate.pp` needs the puppet provisioner, so puppet has to be installed on the guest. It isn't in the `ubuntu/xenial64` base box.